bin/demo.py

```diff
@@ -138,63 +118,84 @@ def update_marathon_json(elb_url):
 def update_and_push_marathon_json(elb_url, branch):
     elb_hostname = strip_to_hostname(elb_url)
     update_marathon_json(elb_hostname)
-    if call (['git', 'commit', '-a', '-m', 'Update marathon.json with ELB URL']) != 0:
-        log ("Failed to commit updated marathon.json.")
-    if call (['git', 'push', 'origin', branch]) != 0:
-        log_and_exit ("Failed to push updated marathon.json.")
-    log ("Updated marathon.json with ELB hostname {}.".format(elb_hostname))
-
-def create_credentials(jenkins_url, id, username, password):
-    credential = { 'credentials' : { 'scope' : 'GLOBAL', 'id' : id, 'username' : username, 'password' : password, 'description' : id, '$class' : 'com.cloudbees.plugins.credentials.impl.UsernamePasswordCredentialsImpl'} }
-    data = {'json' : json.dumps(credential) }
-    headers = auth_func({})
-    post_url = "{}/credential-store/domain/_/createCredentials".format(jenkins_url)
-    r = requests.post(post_url, headers=headers, data=data, verify=False)
-
-def create_job(jenkins_url, job_name, job_config):
-    log ("Creating job")
-    headers = auth_func({'Content-Type' : 'application/xml' })
-    post_url = "{}/createItem?name={}".format(jenkins_url, job_name)
-    r = requests.post(post_url, headers=headers, data=job_config, verify=False)
-    if r.status_code != 200:
-        log ("Failed to create job {} at {}.".format(job_name, jenkins_url))
-        r.raise_for_status()
-    log ("Job {} created successfully".format(job_name))
-
-def create_view(jenkins_url, view_name, view_config):
-    log ("Creating view")
-    headers = auth_func({'Content-Type' : 'text/xml' })
-    post_url = "{}/createView?name={}".format(jenkins_url, view_name)
-    r = requests.post(post_url, headers=headers, data=view_config, verify=False)
+    if call(['git', 'commit', '-a', '-m', 'Update marathon.json with ELB URL']) != 0:
+        log("!! failed to commit updated marathon.json")
```

Comment:
Should this be `log_and_exit`? The next line is a `git push`, but should that happen if this commit fails?